src/main/java/com/woowacamp/soolsool/core/cart/controller/CartItemController.java

```diff
@@ -0,0 +1,33 @@
+package com.woowacamp.soolsool.core.cart.controller;
+
+import com.woowacamp.soolsool.core.cart.code.CartItemResultCode;
+import com.woowacamp.soolsool.core.cart.dto.request.CartItemSaveRequest;
+import com.woowacamp.soolsool.core.cart.service.CartItemService;
+import com.woowacamp.soolsool.global.auth.dto.LoginUser;
+import com.woowacamp.soolsool.global.common.ApiResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/cart-items")
+public class CartItemController {
```

Comment:
```suggestion
@RestController
@RequestMapping("/cart-items")
@RequiredArgsConstructor
public class CartItemController {
```

순서가 불-편했는데 `MemberController` 슨배님은 이렇게 되어있더라구요.

Reply:
LiquorController의 어노테이션도 해당 순서에 맞게 재구성했습니다!!